lib/msal-browser/src/cache/TokenCache.ts

```diff
@@ -46,17 +46,20 @@ export class TokenCache implements ITokenCache {
      * @param request
      * @param response
      * @param options
+     * @returns The homeAccountId of the account associated with the response.
      */
-    loadExternalTokens(request: SilentRequest, response: ExternalTokenResponse, options: LoadTokenOptions): void {
+    loadExternalTokens(request: SilentRequest, response: ExternalTokenResponse, options: LoadTokenOptions): string {
```

Comment:
Instead of just returning a string, I would rather we return an object of some kind (that has homeAccountId as one of its properties, among others), such as `CacheRecord`: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-common/src/cache/entities/CacheRecord.ts#L12

Reply:
Got it